--- osr/src/ogr_p.h_	2015-08-06 07:49:24.000000000 -0700
+++ deps/osr/src/ogr_p.h	2015-08-06 07:50:15.000000000 -0700
@@ -130,33 +130,4 @@
 
 OGRErr CPL_DLL OGRCheckPermutation(int* panPermutation, int nSize);
 
-/* GML related */
-
-OGRGeometry *GML2OGRGeometry_XMLNode( const CPLXMLNode *psNode,
-                                      int bGetSecondaryGeometryOption,
-                                      int nRecLevel = 0,
-                                      int nSRSDimension = 0,
-                                      int bIgnoreGSG = FALSE,
-                                      int bOrientation = TRUE,
-                                      int bFaceHoleNegative = FALSE);
-
-/************************************************************************/
-/*                        PostGIS EWKB encoding                         */
-/************************************************************************/
-
-OGRGeometry CPL_DLL *OGRGeometryFromEWKB( GByte *pabyWKB, int nLength, int* pnSRID,
-                                          int bIsPostGIS1_EWKB  );
-OGRGeometry CPL_DLL *OGRGeometryFromHexEWKB( const char *pszBytea, int* pnSRID,
-                                             int bIsPostGIS1_EWKB );
-char CPL_DLL * OGRGeometryToHexEWKB( OGRGeometry * poGeometry, int nSRSId,
-                                     int bIsPostGIS1_EWKB );
-
-/************************************************************************/
-/*                        WKB Type Handling encoding                    */
-/************************************************************************/
-
-OGRErr OGRReadWKBGeometryType( unsigned char * pabyData,
-                               OGRwkbVariant wkbVariant,
-                               OGRwkbGeometryType *eGeometryType, OGRBoolean *b3D );
-
 #endif /* ndef OGR_P_H_INCLUDED */
